homeassistant/components/aepohio/manifest.json

```diff
@@ -0,0 +1,6 @@
+{
+  "domain": "aepohio",
```

Comment:
This domain is incorrect (and all other too). Spaces in names, should result in an `_` in the integration domain. None of these used that.